Classes/UT3Nightshade.uc

```diff
@@ -91,8 +92,39 @@ var float LastDropAttemptTime;
 var array< class<Actor> > MineObjectClasses;
 /* The mine that's on the arm when we deploy. */
 var Actor ArmMine;
+/* The third person camera distance before we deployed. */
+var float LastDesiredTPCamDistance;
+/* The third person camera distance to use when deployed. */
+var float DeployedTPCamDistance;
+/* The third person camera position to use when deployed. */
+var vector DeployedTPCamLookat;
 
 
+
+simulated function Destroyed() {

```

Comment:
I try to keep the style consistent with UT2004, i.e. every `{` should be on its own line and indents are four spaces rather than a tab.

Reply:
Sorry on this one, habit kicked in on this one!